keadm/cmd/keadm/app/cmd/util/common.go

```diff
@@ -326,7 +326,17 @@ func checkKubernetesVersion(serverVersion *version.Info) error {
 //installKubeEdge downloads the provided version of KubeEdge.
 //Untar's in the specified location /etc/kubeedge/ and then copies
 //the binary to excecutables' path (eg: /usr/local/bin)
-func installKubeEdge(componentType types.ComponentType, arch string, version semver.Version) error {
+func installKubeEdge(options types.InstallOptions, arch string, version semver.Version) error {
+	//create the storage path of the kubeedge installation packages
```

Comment:
Add blank after `//`

Reply:
but I did not change the content here,I noticed that there are still many situations where there is no blank after // ？？？